apps/heureka/src/hooks/useUrlState.js

```diff
@@ -74,7 +80,17 @@ const useUrlState = () => {
     if (JSON.stringify(updatedState) !== JSON.stringify(currentState)) {
       urlStateManager.push(updatedState)
     }
-  }, [loggedIn, detailsFor, activeView])
+  }, [
+    loggedIn,
+    detailsFor,
+    activeView,
+    serviceActiveFilters,
```

Comment:
This useEffect looks confusing to me since there is no relation from the dependency array to the code being used inside in the hook. This could be improved in a new PR. Would you create a task for it?

Reply:
Let's discuss this, since as I said these dependencies are taking care of updating the filter props in the URL. The task that I mentioned in this PR, is in fact for solving this problem, so we do not need a new task, I suppose.